.github/workflows/server-ci.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         SVIX_REDIS_DSN: "redis://localhost:6379"
         SVIX_QUEUE_TYPE: "redis"
         SVIX_JWT_SECRET: "test value"
+        SVIX_CACHE_TYPE: "memory"
```

Comment:
Should this be our default testing approach? I wonder if we should run tests twice for now in CI with `redis` specified as well until we have a better strategy for rolling through the various combinations.

Reply:
Resolving this for now, will take alternate approach to testing in future PR